### DISCUSSION TITLE
For Seeed nRF52 mbed-enabled Boards Boards | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#167] [Q&A]
# For Seeed nRF52 mbed-enabled Boards Boards | Seeed Studio Wiki

For Seeed nRF52 mbed-enabled Boards Boards

https://wiki.seeedstudio.com/XIAO-BLE-Sense-Bluetooth-Usage/

**Comment by ericphilbaz** [comment]
BLE lib
警告： ArduinoBLE-master 库要求运行在 samd, megaavr, mbed, apollo3, mbed_nano, mbed_portenta, mbed_nicla, esp32, mbed_giga 架构()，可能与你现在运行在 nrf52 架构上的开发板()不兼容。

**Comment by sdr-uami** [comment]
Why is not possible to use ArduinoBLE library with this board?

**Reply by MatthewJeffson** [reply]
Hi, this is I think should be the method to use ArduinoBLE library. Have you tried to follow the instructions from the wiki to do it?

**Reply by sdr-uami** [reply]
Yes, I followed the instructions here https://wiki.seeedstudio.com/XIAO-BLE-Sense-Bluetooth-Usage/

Where you claim nrf52840 sense works with ArduinoBLE, but that is not true, the official library does not support this board, which can be seen here: https://reference.arduino.cc/reference/en/libraries/arduinoble/

Please remove that information from the wiki or show how to use nrf52840 sense with that library.

I just bought this board because of the information in the wiki, and not it nos possible to run it.

**Reply by MatthewJeffson** [reply]
Hi, I have installed the official library in the [section](https://wiki.seeedstudio.com/XIAO-BLE-Sense-Bluetooth-Usage/#arduino-library-overview) and it works fine for me.
![企业微信截图_16860313281891](https://github.com/Seeed-Studio/wiki-documents/assets/104187183/3cee62ed-758b-49fa-b501-5a0b4a7c85fe)

**Reply by sdr-uami** [reply]
ArduinoBLE is working, the problem was the board, I had to install NRF52840 2.7.2. thanks for your help
